libs/components/indicators/src/lib/modules/label/label.component.ts

```diff
@@ -15,23 +15,13 @@ const LABEL_TYPE_DEFAULT = 'info';
 export class SkyLabelComponent {
   /**
    * The type of label to display.
-   * @required
+   * @default 'info'
    */
   @Input()
   public set labelType(value: SkyLabelType | undefined) {
-    this.#_labelType = value;
+    this.labelTypeOrDefault = value === undefined ? LABEL_TYPE_DEFAULT : value;
 
-    if (this.#_labelType === undefined) {
-      this.labelTypeOrDefault = LABEL_TYPE_DEFAULT;
-    } else {
-      this.labelTypeOrDefault = this.#_labelType;
-    }
-
-    this.updateIcon();
-  }
-
-  public get labelType(): SkyLabelType | undefined {
```

Comment:
I think you still want to track `#_labelType` and expose it in a `get` accessor. We just need to also set the default value on the `labelTypeOrDefault` in the `set` accessor.

Reply:
We chatted about this and are going to leave as is.